service/azureconfig/v1/resource/deployment/resource.go

```diff
@@ -226,11 +226,6 @@ func (r *Resource) ApplyUpdateChange(ctx context.Context, obj, updateState inter
 	return nil
 }
 
-// Underlying returns the underlying resource.
-func (r *Resource) Underlying() framework.Resource {
-	return r
-}
-
```

Comment:
This wasn't even compiling now. Now `Resource` fulfills `framework.CRUDResourceOps`, not `framework.Resource`.